src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -49,6 +49,12 @@
             <div class="sf-toolbar-info-piece">
                 <span>You are not authenticated.</span>
             </div>
+            {% if collector.firewall %}
+            <div class="sf-toolbar-info-piece">
+                <b>Firewall name</b>
+                <span>{{ collector.firewall.name }}</span>
+            </div>
+            {% endif %}
```

Comment:
It looks like we could actually merge the `collector.token` and `collector.enabled` branch and customise only some of the displayed elements instead.